.github/workflows/build.yaml

```diff
@@ -1,23 +1,73 @@
-name: EDM4U build
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build
+
+on:
+  schedule:
+    - cron: "0 10 * * *"  # 10am UTC = 2am PST
+
+  pull_request:
+    types: [ labeled, closed ]
 
-on: 
   workflow_dispatch:
     inputs:
       unity_version:
         description: 'Unity version'
         default: '2019'
-        required: true
         type: string
+        required: true
 
 env:
-  pythonVersion: '3.7'
-  
-  
+  # Use SHA256 for hashing files.
+  hashCommand: "sha256sum"
+
 jobs:
-  build_desktop:
-    name: build-macOS-unity${{ inputs.unity_version}}
-    runs-on: macos-latest
+  build_macos:
+    name: build-macos-unity${{ inputs.unity_version}}
+    uses: ./.github/workflows/build_macos.yaml
+    with:
+      unity_version: ${{ inputs.unity_version}}
+
+  finalizing:
+    # Only compute SHA hash for macOS build
+    name: finalizing-macOS-unity${{ inputs.unity_version}}
+    needs: [build_macos]
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - run: python -v
-      - run: sysctl -n machdep.cpu.brand_string
+      - name: Fetch All builds
+        uses: actions/download-artifact@v3
+        with:
+          path: built_artifact
+
+      - name: Compute Plugin Hash
+        shell: bash
+        run: |
+          # Compute hash for .tgz package
+          pushd built_artifact/TarballPackage_macOS
+          tgz_files_list=$(find -type f -name '*.tgz')
+          for tgz_file in "${tgz_files_list[@]}"; do
+            echo tgz_file
+            ${{ env.hashCommand }} --tag ${tgz_file} >> edm4u_hash.txt
+          done
+          echo "::warning update sha txt \n ::$(cat edm4u_hash.txt)"

```

Comment:
Do you still want this warning?  If so, it could be clearer what it is trying to say.

Reply:
This seems to be how it shows SHA256 as Annotations.
This is how it was done in C++ and Unity repo as well. Ex.

https://github.com/firebase/firebase-cpp-sdk/blob/86198bb73e4211f66db49c23acd4af63dd3bd1f3/.github/workflows/cpp-packaging.yml#L685